ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -9568,7 +9568,7 @@ def test_torch_instance_addr_(
         allow_inf=False,
     ),
 )
-def test_torch_special_eq_(
+def test_torch___eq___(
```

Comment:
This is testing `eq_`, not `__eq__`. The name of the test was wrong.

Reply:
Done